packages/frontend/core/src/hooks/use-journal.ts

```diff
@@ -36,9 +39,17 @@ export const useJournalHelper = (workspace: BlockSuiteWorkspace) => {
       initEmptyPage(page, title).catch(err =>
         console.error('Failed to load journal page', err)
       );
+      adapter.setJournalPageDateString(page.id, title);
       return page;
     },
-    [bsWorkspaceHelper]
+    [adapter, bsWorkspaceHelper]
+  );
+
+  const isPageJournal = useCallback(
+    (pageId: string) => {
+      return isJournalString(adapter.getJournalPageDateString(pageId));
```

Comment:
Maybe we don't need to validate format which is already stored on `pageInfo`?

Reply:
it does no harm with an additional check while it makes the implementation a bit more robust

Reply:
😆 Yes, I'm just worry about that in case we change the format later, then there could be multiple formats. Anyway, that not matters, let's keep it this way!

Reply:
Is the isJournalString function still needed?